controllers/ibmpowervscluster_controller.go

```diff
@@ -106,7 +106,7 @@ func (r *IBMPowerVSClusterReconciler) Reconcile(ctx context.Context, req ctrl.Re
 	return r.reconcile(clusterScope), nil
 }
 
-func (r *IBMPowerVSClusterReconciler) reconcile(clusterScope *scope.PowerVSClusterScope) ctrl.Result {
+func (r *IBMPowerVSClusterReconciler) reconcile(clusterScope *scope.PowerVSClusterScope) ctrl.Result { //nolint:unparam
```

Comment:
why nolint `nolint:unparam` here?

Reply:
```
controllers/ibmpowervscluster_controller.go:109:90: (*IBMPowerVSClusterReconciler).reconcile - result 0 (sigs.k8s.io/controller-runtime/pkg/reconcile.Result) is always nil (unparam)
func (r *IBMPowerVSClusterReconciler) reconcile(clusterScope *scope.PowerVSClusterScope) ctrl.Result {
```

Reply:
Yes on it, fixing the related test case

Reply:
this needs some more modifications, leave it as is for now with nolint.. will make the modifications..